components/core/App.js

```diff
@@ -19,7 +19,7 @@ const App = () => {
 
   return (
     <>
-      <div className="demo-list">
+      <div className="tw-flex tw-flex-wrap tw-items-center tw-justify-center tw-p-10">
```

Comment:
Padding should be 10px not p-10. p-10 means 2.5rem

Reply:
yes sorry, missed the px.